### PR TITLE
feat: dynamic contracts

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: src/contracts/dev/LedaNFT-address.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "address": "0x5FbDB2315678afecb367f032d93F642f64180aa3"
```
Comment (juliswer) [inline src/contracts/dev/LedaNFT-address.json:2]: Be careful with this

**Review by juliswer — APPROVED** [review]
looks good

File: src/contracts/dev/LedaNFT-address.json (hunk 1 above)
Comment (AlefrankM) [reply src/contracts/dev/LedaNFT-address.json:2]: Actually, I’m building this feature to not be careful with this

**Review by mmiranda-stackit — APPROVED** [review]
very nice

**Pull request merged**